doc/dev-jumpstart.md

```diff
@@ -26,6 +26,23 @@ not stable enough to be moved into `tests`.
 * The `scripts` folder contains several helper scripts for using Mystikos
 or integrating with the pipeline.
 
+Other directories include:
+
+* `apline`: contains Docker image used by some tests
+* `art`: contains graphical resources used in documentation
+* `asm`: contains assembly code
+* `crt`: contains Mystikos's modification to musl library
+* `doc`: contains documentation
+* `docker`: contains file to build Mystikos on Docker
+* `ext2`: contains EXT2 file system implementation
+* `gcov`: contains gcov code coverage tool
+* `host`: contains files used by Mystikos host
+* `hostfs`: a file system not in use
+* `include`: header files
+* `json`: a json parser implementation
+* `oe`: contains extension to OpenEnclave
```

Comment:
@jxyang could you help shed some more light on the purpose of the `oe` folder?

Reply:
Please see my comments below.